README.md

```diff
@@ -190,7 +190,7 @@ additional details on each available environment variable.
 | `ECS_UPDATES_ENABLED` | &lt;true &#124; false&gt; | Whether to exit for an updater to apply updates when requested. | false | false |
 | `ECS_DISABLE_METRICS`     | &lt;true &#124; false&gt;  | Whether to disable metrics gathering for tasks. | false | false |
 | `ECS_POLL_METRICS`     | &lt;true &#124; false&gt;  | Whether to poll or stream when gathering metrics for tasks. Setting this value to `true` can help reduce the CPU usage of dockerd and containerd on the ECS container instance. See also ECS_POLL_METRICS_WAIT_DURATION for setting the poll interval. | `false` | `false` |
-| `ECS_POLLING_METRICS_WAIT_DURATION` | 10s | Time to wait between polling for metrics for a task. Not used when ECS_POLL_METRICS is false. Maximum value is 20s and minimum value is 5s. If user sets above maximum it will be set to max, and if below minimum it will be set to min. | 10s | 10s |
+| `ECS_POLLING_METRICS_WAIT_DURATION` | 10s | Time to wait between polling for metrics for a task. Not used when ECS_POLL_METRICS is false. Maximum value is 20s and minimum value is 5s. If user sets above maximum it will be set to max, and if below minimum it will be set to min. Note that as the number of tasks/containers increased, a higher ECS_POLLING_METRICS_WAIT_DURATION value can potentially cause a problem that reserved memory of ECS cluster become unstable due to missing metrics smaple at metric collection time. The recommendation is to keep this value smaller than 18s. This behavior is also related to OS ad platform. | 10s | 10s |
```

Comment:
Thanks for making this update!

I have some suggested revisions in ***bold italics*** below (address some typos/fix some grammar/improve clarity in some places):
> ***CAUTION: As*** the number of tasks/containers increased, a higher ECS_POLLING_METRICS_WAIT_DURATION value can potentially cause a problem ***where*** reserved memory of ECS cluster ***becomes*** unstable due to missing metrics ***sample*** at metric collection time. ***It is recommended*** to keep this value smaller than 18s. This behavior ***is only observed on certain*** OS ***and platforms***. 

Can we also wrap "ECS_POLLING_METRICS_WAIT_DURATION" here is wrapped with ` character on each side, so that it appears as code markdown?

Reply:
sure!